### ISSUE TITLE
Cropping: allow entering pixels instead of 0 to 1 values

### ISSUE THREAD

**Issue by eugenesvk** [woelper/oculante#87]

**Comment by woelper** [comment]
Very reasonable. I'd like to make everything relative to keep operators resolution-independent, but there should be an option to enter pixel values, too.

**Comment by eugenesvk** [comment]
Yeah, relative is fine (50% is easier than "how many pixels of 1742?")
Potential UI trick to allow both: `.x` (please without any leding `0`) is relative, `y` is pixels, so
`.5` is 50%
`50` is 50 pixels
